docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,109 @@
+---
+id: manage-connector-templates
+title: Manage Connector templates
+description: Manage your Connector templates in Web Modeler.
+---
+
+export const UploadIcon = () => <span style={{verticalAlign: "text-top"}}><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="16" height="16"><path d="M0 0h24v24H0z" fill="none"></path><path d="M9 16h6v-6h4l-7-7-7 7h4zm-4 2h14v2H5z" fill="currentColor"></path></svg></span>;
+
+<span class="badge badge--cloud">Camunda Platform 8 only</span>
+
+You can create and manage [Connector templates](/components/integration-framework/connectors/custom-built-connectors/connector-templates.md) just as any other asset in a Web Modeler project.
+
+## Create Connector templates
+
+Take the following steps to create a new Connector template::
```

Comment:
Minor tweak:
Update text from "Take the following steps to create a new Connector template::"
to "The steps to create a new Connector template are as follows:"